.github/workflows/rustsec-audit.yml

```diff
@@ -0,0 +1,23 @@
+##
+# This GitHub Action is using cargo-audit to perform an audit for crates with security vulnerabilities.
+# https://github.com/rustsec/audit-check
+##
+
+name: Security audit
+on:
+  workflow_dispatch:
+  push:
+    paths:
+      - "**/Cargo.toml"
+      - "**/Cargo.lock"
+  schedule:
+    - cron: "40 13 * * 0"
+jobs:
+  security_audit:
+    runs-on: ${{ fromJson(github.repository_owner == 'subspace' && '["self-hosted", "ubuntu-20.04-x86-64"]' || 'ubuntu-22.04') }}
+    steps:
+      - uses: actions/checkout@v3
+      - uses: rustsec/audit-check@v1.4.1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+        continue-on-error: true
```

Comment:
Why?

Reply:
`continue-on-error: true` needed for scan results? I don't get it

Reply:
It's not needed was left over from when I was testing the security advisories
https://github.com/subspace/subspace/actions/runs/7624889136/job/20769203937